### PR TITLE
Include rule for github.com

### PR DESCRIPTION
**Pull request by ameya-pandilwar**

### CODE REVIEW

**Comment by igor-makarov** [comment]
These look like pretty common-sense requirements that any good password manager adheres to. 

The intention of this repo is to document sites that do not conform to the industry best practices.

**Comment by djbe** [comment]
@igor-makarov the github.com one should be added though, because of the `required: digit` part.

**Comment by ameya-pandilwar** [comment]
Hi @igor-markov -- I agree with what @djbe is saying and we should include `github.com`

File: quirks/password-rules.json [hunk 1]
```diff
@@ -120,6 +120,9 @@
     "girlscouts.org": {
         "password-rules": "minlength: 8; maxlength: 16; required: lower; required: upper; required: digit; allowed: [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789$#!];"
     },
+    "github.com": {
+        "password-rules": "minlength: 8; required: lower; required: digit;"
```
Comment (rmondello) [inline quirks/password-rules.json:124]: Github allows a much larger character set than just lower and digit, so this rule is incomplete. I suspect that Github isn’t really disallowing any specific characters, and their rules seem pretty reasonable, so I’m not sure they’re worth including in this project. (I am totally open to discussion about this point, however!)

Comment (djbe) [reply quirks/password-rules.json:124]: GitHub password rules (https://help.github.com/en/github/authenticating-to-github/creating-a-strong-password):
- Eight characters long, if it includes a number and a lowercase letter, or
- at least 15 characters long with any combination of characters

This is maybe a more general discussion for this project: do we want to document every password quirk for each website? Or can we agree to a set of rules that we **assume** that each password manager will at least adhere to? I assume that each password manager will generate something longer than 8 characters. Not sure if I can assume the same for 16 chars (or longer), that's something that may be dependent on the user's settings.

Comment (rmondello) [reply quirks/password-rules.json:124]: I think we do want to do this! I think we want to define such a policy, document it in the README, and reference it in the PR checklist. I opened https://github.com/apple/password-manager-resources/issues/134 to track this. (I’d be happy for anyone to take a crack at it.)

Comment (ameya-pandilwar) [reply quirks/password-rules.json]: @rmondello -- Updated to include the allowed character set.

File: quirks/password-rules.json [hunk 2]
```diff
@@ -120,6 +120,9 @@
     "girlscouts.org": {
         "password-rules": "minlength: 8; maxlength: 16; required: lower; required: upper; required: digit; allowed: [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789$#!];"
     },
+    "github.com": {
+        "password-rules": "minlength: 8; required: lower; required: digit; allowed: [abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ0123456789`!@#$%^&*()+~{}'\";:<>?];"
```
Comment (rmondello) [inline quirks/password-rules.json:124]: This is equivalent to the easier to read: `minlength: 8; required: lower; required: digit; allowed: lower, upper, digit, [`!@#$%^&*()+~{}'\";:<>?];`

(I should fix the girlscouts.org rule, which is currently setting a bad example.)

Comment (ameya-pandilwar) [reply quirks/password-rules.json]: @rmondello -- Thanks for the suggestion. I fixed it (along with the girlscouts.org rule as well)

**Pull request merged**